### DISCUSSION TITLE
Is there a way to listen to fullscreen control events?

### DISCUSSION THREAD

**Discussion by timmyd87** [TheWidlarzGroup/react-native-video#4266] [Q&A]
### What are you trying to do?

I need to limit the seeking of a video to the position the user has watched up to (we capture this in `onProgress`). I can do it if I implement my own controls, but I want to know if there is a way to listen to fullscreen control / remote events so I can apply the same seek limit in fullscreen.

### What have you tried?

I implemented my own controls, which works. But as soon as we go fullscreen and I seek the video using the fullscreen controls, `onSeek` is not called, and `state.isSeeking` in `onPlaybackStateChanged` remains `false` when seeking — only `isPlaying` is updated to `false`. This happens on iOS; it seems to be OK on Android.

### Platforms

iOS, Android

**Comment by freeboub** [comment]
Yes we already investigated that and it is not possible on ios... I can only propose that you switch to a custom ui for full screen.
It's understood that's a critical use case for ads for exemple ...
